cmd/porter/version.go

```diff
@@ -7,15 +7,18 @@ import (
 )
 
 func buildVersionCommand(p *porter.Porter) *cobra.Command {
-	opts := version.Options{}
+	opts := version.VersionOpts{}
 	cmd := &cobra.Command{
 		Use:   "version",
 		Short: "Print the application version",
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			return opts.Validate()
+			return opts.Options.Validate()
```

Comment:
Please simplify the here back to: 

```suggestion
			return opts.Validate()
```

We can do that because `Options` is embedded in `VersionOpts` (it isn't a named field) so functions available on that type can be called without having to specify that it's coming from `Options`. Apologies if you knew about that, I'm just sharing Go tips in case they are helpful! 😀

Reply:
Thanks, this is all very helpful 🙏🏾